Documentation/contributing/development/reviewers_committers/review_docs.rst

```diff
@@ -0,0 +1,228 @@
+
+Completeness
+~~~~~~~~~~~~
+
+Make sure that new or updated content is complete, with no TODOs.
+
+Auto-generated reference documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are some reference documents (command reference, Helm reference) that
+should be automatically updated when parts of the Cilium repository change. You
+should ensure that everything is up-to-date, although the CI does in fact
+handle the verification for you. But you may have to accompany contributors and
+help them figure out the command they need to run to perform the update.
+
+Spell checker exceptions
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Documentation checks include running a spell checker, which has a list of
+exceptions ("good words"). Team cilium/docs-structure is the owner for this
+file. There's usually not much to look at, but it's good to keep it clean:
+
+  - Entries should remain sorted alphabetically (mind the case)
+  - If a new entry is misspelled, reject it
+  - Lowercase entries are considered case-insensitive, so it's no use adding an
+    entry with capital letters if the lowercase version is already in
+
+Netlify preview
+~~~~~~~~~~~~~~~
+
+Netlify builds a new preview for each PR touching the documentation. You are
+not expected to check the preview for each PR. However, do have a look if the
+PR contains some involved formatting changes, such as nested blocks or
+directives, or work on tables or tabs. Also look at the preview if you have a
+doubt as to the validity of the reStructuredText (RST) mark-up that the author
+uses.
+
+Formatting
+----------
+
+Read :ref:`Cilium's documentation style guide <docs_style_guide>`.
+
+Flag poor formatting or obvious mistakes. The syntax for RST is not always
+trivial and some contributors get it wrong, or they simply forget this is RST
+and they employ Markdown mark-up instead. Make sure they fix such issues.
+
+Keep an eye on :ref:`code-blocks <docs_style_code_blocks>`: do they include RST
+substitutions, and if so, do they use the right directive? If not, do they use
+the right language?
+
+Beyond that, the amount of time you should spend on suggestions for improving
+formatting is up to you.
+
+Grammar and style
+-----------------
+
+Read :ref:`Cilium's documentation style guide <docs_style_guide>`.
+
+Flag obvious grammar mistakes. Try to read the updated text as a user would.
+Ask the contributors to rework any sentence that is too difficult to read or to
+understand.
+
+We aim at keeping the documentation clean, consistent, and in a clear and
+comprehensible state. User experience must always be as good as possible. To
+achieve this objective, Documentation updates should follow best practices,
+such as the ones from the style guide, but reviewing PRs in-depth to flag all
+potential style improvements can be time consuming, so the amount of effort
+that you put into style guidance is up to you.
+
+As of this writing, there is no tooling in place to enforce particular style
```

Comment:
Should we rely on versioning to handle in-time statements?

```suggestion
There is no tooling in place to enforce particular style
```

Reply:
Also see [2]

Reply:
No that's another leftover from the Google Doc, I should just switch to your suggestion.